src/components/TextArea/TextArea.module.css

```diff
@@ -0,0 +1,16 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # TEXTAREA 
```

Comment:
nit: is textarea one word or two words loll 🤔 (doesn't actually need to change)

Reply:
@jinlee93 lol, treating it as one word since [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) does. which may mean the base component name should lower-case the A :)